libs/drawer/src/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { Flex, Heading, Slide, Stack } from '@chakra-ui/react';
+import React from 'react';
+
+import { IconClose } from '@millicast-react/dolbyio-icons';
+import IconButton from '@millicast-react/icon-button';
+
+import { DrawerProps } from './types';
+
+const Drawer = ({
```

Comment:
@jackshen Please add test-id as parameter to the component with the default value